theme/mixins/backdrop.less

```diff
@@ -7,4 +7,5 @@
   background-color: @sk-black;
   opacity: 0.7;
   transition: opacity 0.3s;
+  z-index: @z-index-negative;
```

Comment:
Houston will be the first consumer of that I believe. So we'll test this out, for now, this option won't cause issues for houston as we'll be locking scroll when backdrop is shown. Later we will fix it when we merge some stuff to react-autosuggest and it will be good for everyone